app/controllers/api/v1/organizations_controller.rb

```
@@ -57,6 +64,30 @@ def users
+        unless @organization.users.include?(@user)
+          @organization.users << @user
+          # Send notification to the user
+          NotificationService.notify_user(
+            @user,
+            @organization,
+            "You have been added to the organization: #{@organization.name}"
+          )
+          render json: { message: "User added successfully", user: @user }, status: :created
+        else
+          render json: { message: "User is already a member of this organization" }, status: :ok
+        end
```

RuboCop (1.69.1)

[convention] Do not use `unless` with `else`. Rewrite these with the positive case first.

(Style/UnlessElse)